### ISSUE TITLE
graphqlRequest: cannot run graphql mutation

### ISSUE THREAD

**Issue by manhhailua** [kriasoft/react-starter-kit#1162]
```
Uncaught (in promise) Error: Must contain a query definition.       getFromAst.js:...
```
I merged code from `feature/react-intl` branch and got the above error.

@langpavel : the new way of integrating Apollo-Client broke the whole my projects graph apis. There is currently no `graphqlRequest` for mutation request.
IMO, `graphqlRequest` should support both query and mutation request.

**Comment by langpavel** [comment]
Hi, I'm sorry about that.

Quick fix for you will be to create new helper which will call `apolloClient.mutate`. Just copy [this function](https://github.com/kriasoft/react-starter-kit/blob/feature/apollo/src/store/createHelpers.js#L7..L20) and call it `mutate` for example.

Better, you can use now `import { graphql } from 'react-apollo'; and replace most of your stuff with direct connection to query. I know, this is huge shift, but may be beneficial for you. Consider it.

I'm very interested about this, so please, keep me in :+1:

**Comment by manhhailua** [comment]
@langpavel : Directly using `import { graphql } from 'react-apollo';` may be not beneficial for me I think. I have to track user action logs so redux action is very value to me. So, how can I name every actions with `react-apollo`?
Using `apolloClient.mutate()` ran into another confusing error.
```
Uncaught (in promise) TypeError: Cannot read property 'kind' of undefined          getFromAST.js:24
```

**Comment by langpavel** [comment]
@manhhailua You will need wrap your query into [`gql` template tag](https://github.com/apollographql/graphql-tag#graphql-tag)
```js
// note, this is relatively big dependency and will not be needed if you import *.graphql file directly.
import gql from 'graphql-tag';
const mutationGraphql = gql`mutation { ... }`;
```
or better, you can require `*.graphql` file directly:
```js
import myMutation from './mutationUpdate.graphql'
```
Like this: [src/routes/home/news.graphql](https://github.com/kriasoft/react-starter-kit/blob/feature/apollo/src/routes/home/news.graphql) and [src/routes/home/Home.js](https://github.com/kriasoft/react-starter-kit/blob/feature/apollo/src/routes/home/Home.js#L13)

**Comment by manhhailua** [comment]
@langpavel : here is my current solution (please check this [line of code](https://github.com/manhhailua/adserving/blob/develop/src/store/createHelpers.js#L18)).
`skipCache` option was removed.
Would you mind check [my project `actions` folder](https://github.com/manhhailua/adserving/tree/develop/src/actions) to know how I am using querying.

**Comment by langpavel** [comment]
Ok @manhhailua, It is technically possible to dostinguish query against mutation inside `createGraphqlRequest` so it can be migrated to Apollo completely. I'm marking this as a bug for now, will look at this soon

**Comment by langpavel** [comment]
@jayhuang75 it should by fixed by 08be6ffa095542ae10890e4ce6871ff995390b7e
